index.js

```diff
@@ -76,19 +78,26 @@ if (!output && !dir && !argv.replace) {
 // Use warn to avoid writing to stdout
 console.warn(chalk.bold.red(logo))
 
-spinner.text = `Loading Config`
+spinner.text = 'Loading Config'
 spinner.start()
-Promise.all([ globber(input), getConfig({}, argv.config) ])
-  .then((arr) => {
-    // Until parameter destructuring is supported
-    let files = arr[0]
+getConfig({}, argv.config)
+  .then(() => {
+    spinner.succeed()
 
-    if (!files || !files.length) throw new Error('You must pass a list of files to parse')
+    if (input && input.length) return globber(input)
+    // Read from stdin:
+    else {
+      if (!argv.output) throw new Error('Must specify --output when reading from stdin')
```

Comment:
Why is this line explicitly needed when reading from `process.stdin`? Is L74 not throwing an error anyways if no output is specified? What about letting it write to `process.stdout` instead of throwing an error? That should also enable piping `|` then.

```js
if ( !dir || !output || argv.replace ) chalk.bold.yellow(`
  No Output specified, it's recommended to pass either --output, --dir, or --replace to PostCSS CLI. 
  Result will be written to process.stdout (Console)
`)
```


```bash
postcss index.sss -p 'sugarss' | cleancss
```